### PR TITLE
ReadMe file lacks instructions for postgres - second take

### LINKED ISSUE #17: ReadMe file lacks instructions for postgres
The readme file does not have instructions to create development database and to edit the corresponding file to append their passwords for proper firing of rails server

### PR DESCRIPTION
**Pull request by KrzysztofMadejski**
Fixes #17
Ping @Mr0grog

### CODE REVIEW

**Comment by KrzysztofMadejski** [comment]
Hm, maybe having `DATABASE_URL` as default variable should point to [.env.example file](https://github.com/edgi-govdata-archiving/web-monitoring-db/blob/master/.env.example#L5)? 

The install guide don't mention copying and modifying .env.example. Should it be added in a new advanced section?

**Comment by KrzysztofMadejski** [comment]
Pin @edgi-govdata-archiving/lieutenants

**Comment by patcon** [comment]
LGTM!

I love the idea of at least hat-tipping to advanced stuff, so that, even if the help isn't necessarily comprehensive, new folks don't feel they're going offroading when they hit a slight snag (perhaps for the first time using postgres or something) :)

**Comment by danielballan** [comment]
Looks good. Will let Rob merge when he gets a look.

File: README.md [hunk 1]
```diff
@@ -95,6 +100,47 @@ User.create(
 )
 ```
 
+## Troubleshooting
+
+If you are getting errors such as `FATAL: role "user" doesn't exist. Couldn't create database.` while running `rake db:setup` or `rake db:create` then it may mean that your database is password protected. There are two ways to setup required databases:
+  
+1. (Recommended) Loosen local Postgres database security to allow local users without password
```
Comment (Mr0grog) [inline README.md]: Ok, I may have been a little unclear in other discussions. Sorry for that. I really don’t think we should ever recommend people reduce the security of their existing setup. (I don’t think security is important for a dev DB here, but if someone has any other databases in Postgres that might actually need to be secured, we’re recommending they open up a big vulnerability. That’s bad.)

Can we make this option 2 and make the other (manually create users and DBs) the recommended option?

File: README.md [hunk 2]
```diff
@@ -95,6 +100,47 @@ User.create(
...
+    sudo -u postgres createuser `whoami` -ds
+    ```
+    
+    Now `bundle exec rake db:setup` command should work.
+    
+2. Create users and databases manually
+
+    ```sh
+    sudo -u postgres psql -c "CREATE USER wmdb_dev WITH PASSWORD 'wmdb_dev';"
+    sudo -u postgres createdb -O wmdb_dev wmdb_dev -E utf-8
```
Comment (Mr0grog) [inline README.md]: I totally missed this in the earlier review, but can you please use `web-monitoring-db_development` for the DB name? I’d like to make sure the DB someone winds up with by following this matches as closely as possible to the one they’d wind up with if they did the normal steps (`rake db:setup`).

File: README.md [hunk 3]
```diff
@@ -95,6 +100,47 @@ User.create(
...
+    sudo -u postgres psql -c 'CREATE EXTENSION IF NOT EXISTS "uuid-ossp";' wmdb_dev
+    sudo -u postgres psql -c 'CREATE EXTENSION IF NOT EXISTS "plpgsql";' wmdb_dev
```
Comment (Mr0grog) [inline README.md]: Same here on DB naming.

File: README.md [hunk 4]
```diff
@@ -95,6 +100,47 @@ User.create(
...
+    ```
+    
+    and then set `DATABASE_URL` environment variable to point to the development database:
+     
+    ```sh
+    export DATABASE_URL=postgres://wmdb_dev:wmdb_dev@localhost/wmdb_dev
```
Comment (Mr0grog) [inline README.md]: DB naming again.

**Comment by KrzysztofMadejski** [comment]
@Mr0grog done!

**Comment by Mr0grog** [comment]
Beautiful. Thanks so much for working on this, @KrzysztofMadejski!

**Pull request merged**